Hymns/Display/VerseViewModel.swift

```diff
@@ -22,9 +22,7 @@ class VerseLineViewModel: Hashable, ObservableObject {
         userDefaultsManager
             .fontSizeSubject
             .sink { fontSize in
-                withAnimation(.easeInOut(duration: 0.5)) {
-                    self.fontSize = fontSize
-                }
+                self.fontSize = fontSize
         }.store(in: &disposables)
```

Comment:
I could not find a single animation that results in us not having the ... during transition. To me it's probably better to just forgo the animation all together than have it feed in the odd manner it is now.

Reply:
agreed